packages/e2e-tests/specs/editor/various/copy-cut-paste-whole-blocks.test.js

```diff
@@ -31,6 +31,17 @@ describe( 'Copy/cut/paste of whole blocks', () => {
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 	} );
 
+	it( 'should copy blocks when non textual elements are focused  (image, spacer)', async () => {
```

Comment:
I added a test but the weird thing is that the test pass on trunk even if it fails when you do the same thing manually. I was not able to make the test fail in trunk no matter how I tried to write it.